mobly/controllers/android_device.py

```diff
@@ -712,14 +712,11 @@ def _terminate_sl4a(self):
         the session. Clear corresponding droids and dispatchers from cache.
         """
         if self.sl4a:
-            self.sl4a.closeSl4aSession()
-            self.sl4a.close()
+            self._terminate_jsonrpc_client(self.sl4a)
```

Comment:
Huh? It seems to be working. What do you mean?
ed.clean_up does not send anything to the device side.